recipes/pybammsolvers/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "pybammsolvers" %}
+{% set version = "0.0.4" %}
+{% set sundials_version = "6.5.0" %}
+{% set suitesparse_version = "6.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pybammsolvers-{{ version }}.tar.gz
```

```suggestion
  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Already defined above